sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Diagnostics/LiveMetricsExporterEventSource.cs

```diff
@@ -195,5 +195,20 @@ public void DroppedDocument(DocumentIngressDocumentType documentType)
 
         [Event(12, Message = "Document was dropped. DocumentType: {0}. Not user actionable.", Level = EventLevel.Warning)]
         public void DroppedDocument(string documentType) => WriteEvent(12, documentType);
+
+        [Event(13, Message = "PerformanceCounterCollectorFactory initialized a PerformanceCounterCollector. IsWindows: {0}. IsAzureAppService: {1}. Collector Type: {2}.", Level = EventLevel.Informational)]
```

Comment:
Too generic message, what action to take if we see such message in event source traces

Reply:
EventLevel.Informational.
Not meant to be actionable, this is to audit which type was selected at runtime.